frontend/js/src/pins/PinRecordingModal.tsx

```diff
@@ -111,7 +118,44 @@ export default NiceModal.create(
       },
       [recordingToPin, blurbContent]
     );
-
+    const updatePinnedRecordingComment = React.useCallback(
+      async (event: React.MouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        try {
+          if (rowId && recordingToPin && currentUser?.auth_token) {
+            const response = await APIService.updatePinRecordingBlurbContent(
+              currentUser.auth_token,
+              rowId,
+              blurbContent
+            );
+            if (!response.status) {
```

Comment:
`status` should be an integer of the HTML status code (200, 404, etc) so this condition will always be true.
If the API call fails it will throw an error (see the implementation of `checkStatus` in the APIService file)